### ISSUE TITLE
How to load fallback initramfs?

### ISSUE THREAD

**Issue by kustodian** [andreyv/sbupdate#8]
I set an additional initramfs in sbupdate config like this:
```
INITRD[linux]=/boot/initramfs-linux.img
INITRD[linux]=/boot/initramfs-linux-fallback.img
```
And I noticed that when I added these two lines the signed efi doubled in size, but I have no idea how to boot it with the fallback initramfs. I didn't see a section about this in the README.md.

**Comment by kustodian** [comment]
I want to be able to boot a same kernel with either the default initramfs, or the fallback one. Is there a way to specify that it builds two separate efi images with the same kernel, but different initramfs?

**Comment by andreyv** [comment]
Hi,

Currently there is no support for multiple initramfs per kernel. It is on the TODO list.

If your `/boot` supports symlinks, then, as a workaround, you can create a symlink `/boot/vmlinuz-linux-fallback` pointing to `vmlinuz-linux`. Then it should be automatically matched with `/boot/initramfs-linux-fallback.img`.

Or you can also use `linux-lts` as a fallback.

**Comment by Maryse47** [comment]
@kustodian keep in mind that unless you have some ancient hardware, booting from non-fallback initramfs doesn't have any benefits. You would only waste space.

**Comment by kustodian** [comment]
You're probably right. When I added `autodetect` hook to initramfs it reduced the size of it to about a half and I could feel the difference in boot times, because initramfs loaded faster. But now when I configured secure boot and generated a single efi binary, there isn't much of a difference, because the whole boot process is slower anyway. I'll just remove `autodetect` and it will be the same as fallback.

Thanks for your help and for writing and awesome tool.
